src/lib/connectors/repos/notes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { Note } = require('../bookshelf');
+const helpers = require('./lib/helpers');
+
+const withRelated = [
+  'user'
+];
+
+const create = async data => helpers.create(Note, data);
+
+const update = (noteId, changes) =>
+  Note
+    .forge({ noteId })
+    .save(changes);
+
+/**
+ * Find single note by ID
+ * @param {String} noteId
+ * @return {Promise<Object>}
+ */
+const findOne = async noteId => helpers.findOne(Note, 'noteId', noteId, withRelated);
```

Comment:
with related will not be used but again it can be left for future use

Reply:
I might leave